### PR TITLE
Update testing section for Meteor 1.7

### PR DESCRIPTION
**Pull request by cur33**
The lack of any mention of the new 'testModule' feature in package.json after the Meteor 1.7 release threw me off as a new Meteor user going through the Angular tutorial yesterday, so I updated all three tutorials to reflect the change. Please let me know if my additions should be worded differently, or feel free to make the changes yourself.

### CODE REVIEW

File: content/angular/step11.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ To do so, we'll add a [test driver](http://guide.meteor.com/testing.html#test-dr
 meteor add practicalmeteor:mocha
 ```
 
+> ### New testing feature
+> While the `meteor test ...` command does still work as specified below, the release of Meteor 1.7 includes a new feature which allows you to specify the location of the test module in `package.json` with a property named `"testModule"`, which you can read more about [in the changelog](https://docs.meteor.com/changelog.html#v1720180528). In order to get the expected behavior, such that the `meteor test ...` command only uses files whose names match the format `*.test[s].*` or `*.spec[s].*`, you should either remove the line `"testModule": "tests/main.js"` from your `package.json` file, or change it to an appropriate value, before running `meteor test ...`.
```
Comment (benjamn) [inline content/angular/step11.md]: Should we mention that `"testModule"` need to appear inside a `"meteor"` section of `package.json`?

**Review by benjamn — COMMENTED** [review]
Thanks for addressing this! We should probably document `meteor.mainModule` in the same way, but that doesn't have to be part of this PR.

File: content/angular/step11.md (hunk 1 above)
Comment (cur33) [reply content/angular/step11.md]: That's not a bad idea. I considered it, but the section was already pretty long, so I decided against. I could go either way on it, though.

**Comment by cur33** [comment]
No problem. That's a good idea to include `meteor.mainModule`; I didn't really think to because the presence of `meteor.testModule` was the only one that actually broke the tutorial. I noticed that the Meteor Guide (and likely the API) also need to be updated for these changes, but obviously those are separate repos.

**Comment by cur33** [comment]
@benjamn Added a clarification! Let me know if you think that makes more sense.

**Comment by serut** [comment]
This should be already fixed !!

**Comment by corourke** [comment]
Guys, why has this not been merged in? I just wasted 2 hours on this issue! Ahhhgg!

**Comment by benjamn** [comment]
Thanks for merging @abernix, and thanks to @cur33 for identifying and fixing this problem!

**Comment by cur33** [comment]
You're welcome! I'm glad this finally got added. Thanks to @abernix!

**Pull request merged**